### DISCUSSION TITLE
Issue with pulled vocabularies in 'build' folder.

### DISCUSSION THREAD

**Discussion by joegregoryphd** [opencaesar/platform#22] [General]
When running the 'omlDependencies' task, the default IMCE vocabularies in the 'build' folder of an OML project now contain errors.
This looks like an issue with the subclass notation.

e.g., in the 'XMLschema.oml' file, the following definition is included:
- scalar integer < decimal
but this should be:
 - scalar integer :> decimal
This error propagates through all dependencies.

**Comment by melaasar** [comment]
I think your project dependencies unintentionally picked up the new OML 2.0 dependencies. Make sure your dependencies on core-vocabularies is 3.+ and if you used imce-vocabularies then use 3.+. Find those in the grade build file.

Also in buildScript section make sure all io.opencaesar tools dependencies are 1.+ versions.

**Reply by joegregoryphd** [reply]
Thank you Maged, this resolved the issue.
